model/job/broker.go

```diff
@@ -76,6 +76,7 @@ type (
 	Job struct {
 		JobID       string      `json:"_id,omitempty"`
 		JobRev      string      `json:"_rev,omitempty"`
+		Cluster     int         `json:"cluster,omitempty"`
```

Comment:
This should probably be names `CouchCluster` or `DBCluster` to avoid future collision.